databricks/koalas/indexes.py

```diff
@@ -571,6 +571,33 @@ def __getattr__(self, item: str) -> Any:
     def rename(self, name, inplace=False):
         raise NotImplementedError()
 
+    @property
+    def levels(self) -> list:
+        """
+        Names of index columns in list. Be aware of the possibility of running into out
+        of memory issue if returned list is huge.
+
+        Examples:
+        --------
+        >>> mi = pd.MultiIndex.from_arrays((list('abc'), list('def')))
+        >>> mi.names = ['level_1', 'level_2']
+        >>> kdf = ks.DataFrame({'a': [1, 2, 3]}, index=mi)
+        >>> kdf.index.levels
+        [['a', 'b', 'c'], ['d', 'e', 'f']]
+
+        >>> mi = pd.MultiIndex.from_arrays((list('bac'), list('fee')))
+        >>> mi.names = ['level_1', 'level_2']
+        >>> kdf = ks.DataFrame({'a': [1, 2, 3]}, index=mi)
+        >>> kdf.index.levels
+        [['a', 'b', 'c'], ['e', 'f']]
+        """
+        scols = self._kdf._internal.index_scols
+        sdf = self._kdf._sdf.select([F.collect_set(scol) for scol in scols]).collect()[0]
```

Comment:
nit: `row = self._kdf._sdf.select([F.collect_set(scol) for scol in scols]).first()`

Reply:
looks nicer!! thanks! ^^